src/controllers/main/main.ts

```diff
@@ -471,19 +471,46 @@ export class MainController extends EventEmitter {
   async updateSelectedAccountPortfolio(forceUpdate: boolean = false) {
     if (!this.accounts.selectedAccount) return
 
-    // pass the accountOps if any so we could reflect the pending state
-    const accountOps =
-      this.actions.currentAction?.type === 'accountOp' && this.signAccountOp
-        ? {
-            [this.actions.currentAction.accountOp.networkId]: [this.actions.currentAction.accountOp]
+    const account = this.accounts.accounts.filter(
+      (a) => a.addr === this.accounts.selectedAccount
+    )[0]
```

Comment:
We can use `find` instead of `filter` and we'll get the account directly, skipping the [0]